source/includes/_security.md

```diff
@@ -364,8 +433,25 @@ $userRoles = $apiInstance->getUserRoles($username);
   ]
 }
 ```
+
+**Query Parameters**
+
+None
+
+**Response**
+
+If successful, returns a status code of 200 and a userRoles object.
+
 ### Update roles associated to a user
 
+Updates the Roles associated with a User.
```

Comment:
```suggestion
Updates the roles associated with a user.
```

Reply:
I'm unable to review the last few endpoints probably since these were added earlier. Looks like there is a typo below, it should be `Update an existing role definition`:
<img width="689" alt="image" src="https://github.com/killbill/slate/assets/85998496/62ef1c5c-1856-4968-b7df-59f7125a8af7">